react/features/whiteboard/constants.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Fixed name of the whiteboard fake participant.
+ */
+export const WHITEBOARD_PARTICIPANT_NAME = 'Whiteboard';
```

Comment:
This needs to be translatable.

Reply:
Hm this would mean to translate the participant name wherever it shows up. Are we ok with that?

Reply:
Yes. The ID might be a constant like `whiteboard`, but the default display name needs to be translatable.